java/runtime/src/main/java/org/ray/runtime/WorkerContext.java

```diff
@@ -52,13 +52,15 @@ public WorkerContext(WorkerMode workerMode, UniqueId driverId, RunMode runMode)
     currentTask = ThreadLocal.withInitial(() -> null);
     currentClassLoader = null;
     if (workerMode == WorkerMode.DRIVER) {
-      workerId = driverId;
+      // TODO(qwang): Assign the driver id to worker id
+      // once we treat driver id as a special worker id.
+      workerId = jobId;
```

Comment:
I don't think we should set it to random id now.

In the one of the next several PRs, we should embedding job id to worker id if the mode is DRIVER. Because we should use the variable `workerId` represent the driver id or the worker id in different mode.